client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
+      <li class="static">{{ "Reporting Group" | translate }}</li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <a  class="btn btn-default pull-right"
+          ui-sref="emailreport"
+          ui-sref-opts="{notify:false}">
+            <i class="fa fa-plus"></i>
+            <span  translate=""  class="ng-scope">
+              Email Report
+            </span>
+        </a>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+
+
+<div class="flex-content">
+  <div class="container-fluid" id="anchor">
+
+  <div class="col-lg-12">
+    <!-- start new Reporting group form -->
+    <div class="col-lg-5">
+    <h4> {{"New reporting group"}}</h4>
+
+
+    <form
+      name="ReportGroupRegistrationForm"
+      bh-submit="ReportGroupCtrl.save(ReportGroupRegistrationForm)"
+      bh-form-defaults
```

Comment:
You do not need to include `bhFormDefaults`.  It was deprecated (https://github.com/IMA-WorldHealth/bhima-2.X/issues/1447) and then removed (https://github.com/IMA-WorldHealth/bhima-2.X/issues/1739).